pkg/kubeconfig/kubeconfig.go

```diff
@@ -115,6 +115,9 @@ func ForTokenBased(clusterName, clusterID, host, token string) (string, error) {
 }
 
 func ForClusterTokenBased(cluster *managementv3.Cluster, clusterID, host, token string) (string, error) {
+	if cluster.LocalClusterAuthEndpoint == nil {
```

Comment:
q: do we ever reach here? we get to cluster token logic only if endpoint is enabled

Reply:
no, I added this just in case the function is used else where but I think it is unnecessary. will remove.

Reply:
no, I added this just in case the function is used else where but I think it is unnecessary. will remove.